plugins/runtime/html/overview.html

```diff
@@ -0,0 +1,48 @@
+<div class="runtime-controller"
```

Comment:
- I don't think Expand/Collapse sections are required here. Normal heading would be just fine.
- It might be easier for users to read the property values if we show them in a list instead of a paragraph of text. We could list the values inside a `<dl class="dl-horizontal">` element.
- Do we need the copy to clipboard functionality on everything? The command line might make sense but I'm not sure about the other ones.
- We use a copy to clipboard button on other pages, such as the JMX attributes/operations. Please check the examples [here](http://ui.hawt.io/components/clipboard?main-tab=hawtio-ui-test-pages%2Fcomponents&sub-tab=hawtio-ui-test-pages%2Fcomponents-clipboard).